content/docs/tasks/security/multicluster.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Multicluster PKI
+description: Shows how to use Citadel to construct a multicluster PKI.
+
+weight: 90
+
+---
+{% include home.html %}
+
+This task shows how to use Citadel to construct a PKI
+([public key infrastructure](https://en.wikipedia.org/wiki/Public_key_infrastructure))
+that works across multiple Istio clusters.
+In this architecture, one Citadel runs in a dedicated Kubernetes cluster acts as the root CA.
+It issues and rotates intermediate CA certificates for Citadel instances in each Istio workload cluster.
+Each Istio cluster has a Citadel issuing certificates to workloads in the cluster.
+
+This is an alpha feature and currently it has the following caveats:
+
+* CA certificate revocation is not supported.
+* [X.509 certificate name constraints](https://tools.ietf.org/html/rfc5280#section-4.2.1.10) are not supported.
+* This architecture is subject to change in the future.
+* This task does not solve the multicluster service discovery and secure naming probblems, so extra work is needed
+  to support direct workload-to-workload communication across clusters.
+
+Citadel acting as intermediate CA can automatically communicate with an upstream Citadel to rotate its signing certificate.
+In this task, you will set up a two-level PKI involving one Citadel as the online root CA and one
+Citadel as the intermediate CA.
+The architecture shown in this task can be extended to have multiple levels of intermediate CAs,
+and each upstream Citadel can provision certificates for multiple downstream Citadels.
+
+* Note: This task offers a different solution than the
+  [multicluster installation instructions]({{home}}docs/setup/kubernetes/multicluster-install.html).
```

Comment:
Remove all {{home}} in the doc, no longer necessary.
When referencing content within istio.io, replace all .html suffixes with a single /